### ISSUE TITLE
Mercury - Messenger shade layer 

### ISSUE THREAD

**Issue by micahnyc** [OpenSpace/OpenSpace#1815]
There maybe is an issue with the multiply setting? My note says "Messenger shade multiply; nothing should be multiply"

**Comment by micahnyc** [comment]
another note on this says "multiply shouldn't go blank"

**Comment by alexanderbock** [comment]
I think what it means is that the multiply value might be applied to the alpha component as well, meaning that the planet gets transparent with a multiply setting? 🤔

**Comment by Roxeena** [comment]
I see this behavior that Alex talks about for the Renderable Images too. For example the dots for the Lagrange points in the jwst profile. I would expect only the color to change and not the alpha value

![color-dot](https://user-images.githubusercontent.com/16215103/149923651-3d065f21-13b4-4b83-b658-8ae692fc83b2.gif)

**Comment by WeirdRubberDuck** [comment]
It might have to do with the blending. In several of our renderables, if we set a color that is very dark, the object often becomes close to invisible

**Comment by WeirdRubberDuck** [comment]
@Roxeena that behavior might have to do with the additive blending. The multiply color is only applied to the RGB component, but due to our blending method the final color will look more "transparent" the darker the color is. 

I believe I've seen this problem for multiple renderables that support transparency.

**Comment by micahnyc** [comment]
Yeah for planets, its meaning that the entire planet is going transparent while the tiles for the layer marked multiply are loading. I guess it's multiplying by 0....

**Comment by alexanderbock** [comment]
1. Expected behavior for the RenderableImages
2. The planet should not become transparent when multiplying with low numbers
